core/lib/spree/core/search/product_filter.rb

```diff
@@ -0,0 +1,27 @@
+module Spree
+  module Core
+    module Search
+      class ProductFilter
```

Comment:
maybe `ProductFilters` if it returns a list of filters? or maybe a public method `all` or `available` which return the content of `@filters`?